### ISSUE TITLE
Change export format of openIOC to support openIOC v1.1

### ISSUE THREAD

**Issue by elhoim** [MISP/MISP#620]
Change export format of openIOC to support openIOC v1.1.
OpenIOC v1.1 has been around for two years now.
Things that might help: [the Mandiant repo with more details](https://github.com/mandiant/OpenIOC_1.1) and [a python script to tranform openIOC v1.0 files to v1.1](https://github.com/mandiant/ioc_writer/blob/master/examples/10_to_11_upgrade/openioc_10_to_11.py)

**Comment by iglocska** [comment]
Think this will be more interesting once we move to the new backend as right now it would have to be rather limited either way

**Comment by elhoim** [comment]
Indeed, adding dependency to https://github.com/MISP/MISP/issues/27 to try to keep track

**Comment by adulau** [comment]
This should be as a library at some point and then as an export module. @Rafiot has some ideas to make it better

**Comment by jaegeral** [comment]
any update on that, think it would be pretty handy to have latest format.

**Comment by adulau** [comment]
OpenIOC is nowadays rarely used. I think we can close this issue.
